extensions/tiled-assets/examples/example-dimension.json

```
@@ -0,0 +1,239 @@
+{
+    "stac_version": "0.8.0",
+    "stac_extensions": [
+        "eo",
+        "tiled-assets"
+    ],
+    "id": "sen2l2a-cog-converted",
+    "type": "Feature",
+    "bbox": [
+        14.326171874999998, 48.603515625, 14.501953125000002, 48.33984374999999
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    14.326257705688475,
+                    48.33992958068847
+                ],
+                [
+                    14.501867294311525,
+                    48.33992958068847
+                ],
+                [
+                    14.501867294311525,
+                    48.60342979431152
+                ],
+                [
+                    14.326257705688475,
+                    48.60342979431152
+                ],
+                [
+                    14.326257705688475,
+                    48.33992958068847
+                ]
+            ]
+        ]
+    },
+    "properties": {
+        "datetime": "2018-01-01T00:00:00Z",
+        "start_datetime": "2018-01-01T00:00:00Z",
+        "end_datetime": "2018-12-31T23:59:59Z",
+        "eo:gsd": 10,
+        "eo:platform": "sentinel-2",
+        "eo:instrument": "msi",
```

```suggestion
        "eo:instruments": [
            "msi"
        ],
```
Field was moved to core and is an array now.

---

Reading the spec I that should be "instruments" not "eo:instruments" (as in common metadata).

---

Indeed.